demo/README.md

```diff
@@ -1,44 +1,59 @@
-# demo
+# Pelorus Demo
 
-Scripts to demonstrate pelorus functionality live here.
+The following will provide instructions to install and configure Pelorus specifically for the use of this demonstration. 
+This demonstration will make use of a Tekton pipeline to mimic the development of an application and the subsequent deployments.  This is done to populate Pelorus with data.  The Tekton pipeline pulls source code, builds an example python application in OpenShift and deploys the application.
 
-## demo-tekton.sh
+The details of each build type and the steps required to support Pelorus can be found in the [tekton manifest](./tekton-demo-setup/05-build-and-deploy.yaml)
 
-The following will provide a Tekton pipeline that can build an example python application in various ways. The tasks additionally provide the required metadata for Pelorus and launch and test the sample application.
 
-The details of each build type and the steps required to support Pelorus can be found in the [tekton manifest](./tekton-demo-setup/03-build-and-deploy.yaml)
+Instructions:
 
+* Install the Pelorus operator, using the installation [documentation](../docs/GettingStarted/Installation.md)
 
-Instructions:
+* Fork the [konveyor/pelorus](https://github.com/konveyor/pelorus) git repository to your github organization.
 
-* Fork the konveyor/pelorus git repository to your github organization.
+* Add or configure your github-secret with your Github token.  Instructions can be found in the [Authentication to Remote Services](../docs/GettingStarted/configuration/PelorusExporters.md#authentication-to-remote-services) section of the documentation.
 
-* Add or configure your github-secret with your Github token
+Example:
```

Comment:
| would rephrase here to something like 

"
Add or configure your github-secret with your Github token, by running
```
oc create secret generic github-secret --from-literal=TOKEN=ghp_<snip> -n pelorus
```
Instructions can be found in the [Authentication to Remote Services](../docs/GettingStarted/configuration/PelorusExporters.md#authentication-to-remote-services) section of the documentation.
"